plugins/web-worker-offloading/hooks.php

```diff
@@ -2,7 +2,7 @@
 /**
  * Hook callback for Web Worker Offloading.
  *
- * @since n.e.x.t
+ * @since 0.1.0
```

Comment:
can we update this to 1.0.0 when we release? I like starting at 1.

Reply:
I think `0.1.0` is good specifically because this is an experimental plugin and other experimental plugins (except for `auto-sizes`) are also starting at 0.1.0.